WF_Proton_Launcher/Main.cpp

```diff
@@ -147,6 +150,44 @@ int main(int argc, char** argv)
 	Hashlist hashes;
 	std::vector<std::string> diff;
 
+	if (firstrun) {
+		std::cout << std::endl << "Installing Direct X...";
+
+		FILE* fp;
+		fopen_s(&fp, "directx_Jun2010_redist.exe", "wb");
+		if (!fp)
+		{
+			std::wcout << "Failed to create file on disk." << std::endl;
+		}
+		CURL *curl = curl_easy_init();
+
+		FileWrapper wrap{ fp, {} };
+		curl_easy_setopt(curl, CURLOPT_URL, ("https://download.microsoft.com/download/8/4/A/84A35BF1-DAFE-4AE8-82AF-AD2AE20B6B14/directx_Jun2010_redist.exe"));
```

Comment:
Any particular reason for the superfluous parentheses?

Reply:
ah no reason, accidental. Also I just noticed a bug regarding the Launcher.exe change you made earlier. will post a comment on it on the commit